src/js/jsx/tools/GuidesOverlay.jsx

```diff
@@ -60,17 +61,27 @@ define(function (require, exports, module) {
             var flux = this.getFlux(),
                 applicationStore = flux.store("application"),
                 toolStore = flux.store("tool"),
+                uiStore = flux.store("ui"),
                 modalState = toolStore.getModalToolState(),
+                uiState = uiStore.getState(),
                 currentTool = toolStore.getCurrentTool(),
                 currentDocument = applicationStore.getCurrentDocument();
 
             return {
                 document: currentDocument,
                 tool: currentTool,
-                modalState: modalState
+                modalState: modalState,
+                uiState: uiState
```

Comment:
It seems weird that you are adding these to two state properties just to be used in SCU.  How are these two states accessed within the rendering?